server/splunk/splunk.go

```diff
@@ -161,3 +172,21 @@ func newSplunk(api PluginAPI, st store.Store) *splunk {
 
 	return s
 }
+
+func extractUserInfo(id string) (string, string, error) {
+	if id == "" {
+		return "", "", errors.New("Arguments to extract username and/or token must be 2")
```

Comment:
Can we have the error message be more user friendly, like "please provide username and token like so: `username/token`". It's weird because we accept two different forms based on if it's the first time so idk

Reply:
Done!